workspaces/src/lib.rs

```diff
@@ -16,6 +16,7 @@ pub mod network;
 pub mod operations;
 pub mod prelude;
 pub mod result;
+pub mod serde;
```

Comment:
This module is going to collide with serde

Reply:
right, how about `deser` then, and then we can have `deser::serde_str`?

Reply:
I mean, this type of utility doesn't seem to be that necessary to be duplicated in workspaces. It's basically https://docs.rs/serde_with/1.13.0/serde_with/rust/display_fromstr/index.html and it might just be best for a lib like this to not be opinionated

Reply:
ahh didn't know that existed. I'll just replace that then and remove this